app/extensions/elasticsearch/__init__.py

```diff
@@ -44,6 +44,8 @@
 # Global object session
 session = None
 
+ELASTICSEARCH_VERBOSE = False
+
```

Comment:
Elasticsearch trace is extremely verbose. While some of this may be useful for debugging issue seen in ES (of which there have been a few and they have been tough to debug) it should not be in by default.  
The flask ES trace is already excessive, this just makes it worse.